examples/computer_administrators_script.py

```diff
@@ -0,0 +1,21 @@
+from subprocess import Popen
+from pywinauto import application
+import time
+
+Popen(['compmgmt.msc'], shell = True)

```

Comment:
I think the script has to check if it runs as a privileged user with Administrator rights.
```python
from win32com.shell import shell
if shell.IsUserAnAdmin():
    ....
```
http://stackoverflow.com/questions/2946746/python-checking-if-a-user-has-administrator-privileges/10671675#10671675